kolibri/core/templatetags/kolibri_tags.py

```diff
@@ -39,25 +26,24 @@ def content_renderer_frontend_async():
 
 
 @register.simple_tag()
-def kolibri_main_navigation():
+def initialized_data():
```

Comment:
@MCGallaspy - note some refactoring here to try and make this tag more general and re-use an existing global variable. let me know if you think this is problematic.